trade-finance-manager-api/src/constants/login-results.constant.js

```diff
@@ -1,6 +1,5 @@
 module.exports = {
-  userNotFound: 'User not found',
-  incorrectPassword: 'Incorrect password',
-  userIsDisabled: 'User is disabled',
+  userOrPasswordIncorrect: 'username or password incorrect',
```

Comment:
It's a bit picky, but I'd prefer `usernameOrPasswordIsIncorrect: 'username or password is incorrect'` instead here, to match the ones below